doc/project/project_roles.rst

```diff
@@ -123,6 +123,9 @@ in addition to those listed for Contributors and Collaborators:
 * Responsibility to ensure all contributions of the project have been reviewed
   within reasonable time.
 * Responsibility to enforce the code of conduct.
+* Responsability to triage static analysis issues in their code area. Improper
+  triage may lead hidden defects and potential security risks.
```

Comment:
nit: "lead" sounds weird. maybe "result" would be better? 
In any case, this would be the only point with a listed motivation, so maybe that whole sentence does not belong here?

Reply:
I agree with dropping the sentence. Alternatively, you could add this motivation to the static analysis audit team.